app/models/document.rb

```diff
@@ -64,7 +64,7 @@ def tag_metadata
 
   def tag_labels_for(key)
     Array(attrs.fetch(key, []))
-      .map { |tag| tag.fetch("label") }
+      .map(&method(:get_metadata_label))
```

Comment:
Of course. Agree with @edds comment, better to call the method directly rather than pass it in as a `Method` object.